README.md

```diff
@@ -7,6 +7,8 @@ On the first call to the function the decorator runs it with the provided parame
     from memoize.memoized import Memoized
     import time
 
+    # Memoized.readCache = False
+
     @Memoized(cacheDir = "/home/username/tmp") # cacheDir defaults to /tmp in Linux 
```

Comment:
Remove the spaces around the `=` sign ([PEP 8](https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value)).